subsys/bootloader/Kconfig

```diff
@@ -119,17 +119,18 @@ menuconfig IS_SECURE_BOOTLOADER
 if IS_SECURE_BOOTLOADER
 
 config PM_PARTITION_SIZE_PROVISION
-	hex
-	default 0x0 if SOC_NRF9160
-	default 0x1000 if !SOC_NRF9160
-	prompt "Flash space reserved for PROVISION" if !SOC_NRF9160
+	int
+	default 0 if SOC_NRF9160 || SOC_NRF5340_CPUAPP
+	default FPROTECT_BLOCK_SIZE if !(SOC_NRF9160 || SOC_NRF5340_CPUAPP)
```

Comment:
no need to add if here -> first matched default is always used for kconfig.

Reply:
Thanks, fixed.